tests/Backend/Workspaces/WorkspacesSnowflakeTest.php

```diff
@@ -867,17 +867,35 @@ public function testTableLoadAsView(): void
             'languages',
             new CsvFile($importFile)
         );
+        $aliasId = $this->_client->createAliasTable($bucketId, $tableId, 'languages-alias');
```

Comment:
```suggestion
        // test view IM does not support filters
        $aliasId = $this->_client->createAliasTable($bucketId, $tableId, 'languages-alias');
```

Reply:
jde jen o to, ze ten test je strasne dlouhej kvuli slozitymu initu a tak jednotlive testcasy jsou aspon uvozene komentarem

Reply:
přidal sem komentář